markdown/org/docs/sewing/knit-binding/en.md

```diff
@@ -6,21 +6,22 @@ title: Knit binding
 
 Knit binding is a general term for a type of finish (typically) found on the neck or arm opening of T-shirts and other knitwear.
 
-There's two varieties of this finish, a knit band, and actual knit binding.
+There are two varieties of this finish, a knit _band_ and actual knit _binding_.
 
 ## Knit band
 
-![Like most T-shirts, the one this girl is wearing has a knit band at the neck opening.](knit_band.jpg)
+![Like most T-shirts, the one this woman is wearing has a knit band at the neck opening.](knit_band.jpg)
 
-Knit band is what you typically find at the neck opening of your T-shirt. It's a strip of fabric that is folded double and attached to the edge of the garment.
+Knit band is what you typically find at the neck opening of your T-shirt. It's a strip of fabric that is folded double and attached to the edge of the garment
+such that some of the fabric strip extends beyond the unfinished edge of garment.
 
 This leaves an unfinished edge on the inside that is often sewn down by edge-stitching around the knit band.
 
 ### Knit binding
 
-![This girl's grey undershirt and white T-shirt both have a knit binding at the neckline.](knit_binding.jpg)
+![This woman's grey undershirt and white T-shirt both have a knit binding at the neckline.](knit_binding.jpg)
```

Comment:
```suggestion
![Both T-shirts have a knit binding at the neckline.](knit_binding.jpg)
```

Reply:
I changed the wording to "This grey undershirt and white T-shirt both" (since apparently one of the shirts isn't a T-shirt, technically).